charts/auth-layer-server/README.md

```diff
@@ -0,0 +1,208 @@
+# HederaTheGraph (HTG) Authentication Layer Server
+
+The Authentication Layer Server is a KeyCloak server that provides authentication and authorization services for the HederaTheGraph (HTG). It is based on the [KeyCloak](https://www.keycloak.org/) open source identity and access management solution. 
+With the following configurations:
+- Realm preconfigured with the HTG (htg-auth-layer) client
+- Groups and Roles preconfigured for the HTG client
+- Custom claim mappers for the HTG client
+- Token expiration time set to 10 hours (plenty of time for a one day work session)
```

Comment:
nit: if there are links to docs explaining these settings we're overriding/modifying that would be useful here.

Reply:
yes there is an official documentation for keycloak that we can link here.

Reply:
✅  Done